projects/kit/components/input-phone-international/input-phone-international.component.ts

```diff
@@ -90,13 +98,37 @@ export class TuiInputPhoneInternational extends TuiControl<string> {
     @ViewChild(MaskitoDirective, {read: ElementRef})
     private readonly input?: ElementRef<HTMLInputElement>;
 
+    @ViewChildren('button', {read: ElementRef})
+    private readonly buttons?: QueryList<ElementRef<HTMLButtonElement>>;
+
+    protected readonly tuiFlagPipe = inject(TuiFlagPipe);
+    protected readonly tuiGetCountryCallingCodePipe = inject(
+        TuiGetCountryCallingCodePipe,
+    );
+
     protected readonly dropdown = tuiDropdown(null);
     protected readonly options = inject(TUI_INPUT_PHONE_INTERNATIONAL_OPTIONS);
     protected readonly size = inject(TUI_TEXTFIELD_OPTIONS).size;
     protected readonly open = tuiDropdownOpen();
     protected readonly names = toSignal(inject(TUI_COUNTRIES));
     protected readonly metadata = toSignal(from(this.options.metadata));
     protected readonly countryIsoCode = signal(this.options.countryIsoCode);
+    protected readonly icons = inject(TUI_COMMON_ICONS);
+    protected readonly internationalSearchLabel$ = inject(TUI_INTERNATIONAL_SEARCH);
+
+    // Full list of countries
+    protected readonly countriesSignal = computed<TuiCountry[]>(() =>
+        this.getTuiCountriesFromIsoCodes(this.countries),
+    );
+
+    // Countries list after filtering
+    protected readonly countriesFilteredSignal = computed(() =>
```

Comment:
Let's just call it `filtered` and do one `computed` to get everything, no need to break it into `countriesSignal` and 2 utilities:

```ts
    protected readonly filtered = computed(() =>
        this.countries
            .map((iso) => ({
                iso,
                name: this.names()?.[iso] || '',
                code: tuiGetCallingCode(iso, this.metadata()),
            }))
            .filter(({name, code}) => TUI_DEFAULT_MATCHER(name + code, this.search())),
    );
```